index.html

```diff
@@ -57,18 +65,19 @@ <h1>
                 </ul>
             </div>
             <div class="action">
-                <img src="Assets/Disabled.png" alt="">
+                <img src="Assets/Disabled.png" alt="action">
             </div>
 
         </div>
     </section>
     <!-- Works -->
-    <section class="works">
+    <section id="portfolio" class="works">
         <div class="works_content">
             <h3>My Recent Works</h3>
-            <img src="Assets/Indicator.png" alt="">
+            <img src="Assets/Indicator.png" alt="Indicator">
         </div>
         <div class="works_cards">
```

Comment:
Kindly implement the following changes in the works section of your portfolio  so it looks close to have your design as close as possible to the given template to depict good understanding of project rrquirement and attention to details 👍 :

- [ ]  for the desktop view of the portfolio, the ```horizontal line` after the title ` My Recent Works` should be ommitted, please implement this for your portfolio
- [ ] Please I'd suggest you use the appropriate font color as shown in the Figma design for the text `Multi-Post Stories Gain+Glory`
- [ ] Also, work on the `technologies` part of this section to use the right font colors, margins, padding and box design so it looks as close as possible to the design
- [ ] Lastly, implement the right font-property and padding for the button in this section

Figma design :
![image](https://github.com/GhausAliKhan/My-Portfolio/assets/20537712/a709baba-c239-4e0f-bd9a-c4cab0b8e4f8)

Your design :
![image](https://github.com/GhausAliKhan/My-Portfolio/assets/20537712/95407544-852d-44fe-9966-1b9deecf8ccb)